lib/Container.js

```diff
@@ -197,7 +197,7 @@ Container.prototype.create = Promise.promisify(function(done) {
       `COMMIT_REF=${this.build.commit.ref}`,
       `COMMIT_LINK=${this.build.commit.htmlUrl}`,
       `PWD=${SRC_DIR}`,
-      `PROBO_ENVIRONMENT=TRUE`,
+      'PROBO_ENVIRONMENT=TRUE',
```

Comment:
The changes in the strings format were automatically made by eslint. Apparently, using ES6 templates without variables in them is simply unnecesary/violates coding standards.